docs/source/en/model_doc/siglip.md

```diff
@@ -29,6 +29,7 @@ The abstract from the paper is the following:
 - Usage of SigLIP is similar to [CLIP](clip). The main difference is the training loss, which does not require a global view of all the pairwise similarities of images and texts within a batch. One needs to apply the sigmoid activation function to the logits, rather than the softmax.
 - Training is not yet supported. If you want to fine-tune SigLIP or train from scratch, refer to the loss function from [OpenCLIP](https://github.com/mlfoundations/open_clip/blob/73ad04ae7fb93ede1c02dc9040a828634cb1edf1/src/open_clip/loss.py#L307), which leverages various `torch.distributed` utilities.
 - When using the standalone [`SiglipTokenizer`] or [`SiglipProcessor`], make sure to pass `padding="max_length"` as that's how the model was trained.
+- A basic prompt template of "This is a photo of {label}." should be used. This is handled automatically in Pipelines.
```

Comment:
I don't think this is quite right - any prompt could be used. It's just that this prompt will provide the numbers in the example. 

In the pipeline, it's only handled in the zero-shot case, AFAIK and can be overridden, this is just a default setting.

Reply:
Ok, I think I will rephrase to: "To get the same results as the pipeline, a basic prompt template of "This is a photo of {label}." should be used."
I was thinking more of fine-tuning cases and its weird that you deviate from the pretrained prompt format when finetuning.